### DISCUSSION TITLE
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading '_c') at Proxy.o

### DISCUSSION THREAD

**Discussion by hakuna0829** [quasarframework/quasar#12241] [CLI - PWA mode]
I've installed this package(vue-json-editor) in the vue quasar project.
https://github.com/Kassaila/vue-json-editor
Here is the dependency list.

```
"dependencies": {
    "@kassaila/vue-json-editor": "^0.4.0",
    "@quasar/extras": "^1.0.0",
    "core-js": "^3.6.5",
    "lodash": "^4.17.21",
    "quasar": "^2.0.0",
    "vue-i18n": "^9.0.0",
    "vuex": "^4.0.1"
  },
  "devDependencies": {
    "@babel/eslint-parser": "^7.13.14",
    "@quasar/app": "^3.0.0",
    "eslint": "^7.14.0",
    "eslint-config-standard": "^16.0.2",
    "eslint-plugin-import": "^2.19.1",
    "eslint-plugin-node": "^11.0.0",
    "eslint-plugin-promise": "^5.1.0",
    "eslint-plugin-vue": "^7.0.0",
    "eslint-webpack-plugin": "^2.4.0"
  }
```
I've followed readme like this.
```
<template>
  <json-editor :data-input="dataInput" @data-output="(data) => (dataOutput = data)"> </json-editor>
</template>
<script>
  import JsonEditor from '@kassaila/vue-json-editor';

  export default {
      ...
      components: {
        JsonEditor,
      },
      data() {
          return {
              dataInput: {
                name: 'Taras',
                age: 28,
                address: [
                  'Ukraine',
                  {
                    city: 'Kharkiv',
                  },
                ],
                others: {
                  id: 1246,
                  time: new Date(),
                  description: 'last example run',
                  pattern: /abc/g,
                },
              },
              dataOutput: {},
          };
      },
  }
</script>
```
But it occurs the TypeError like the title.
Is there anyone who help to solve this issue?
Thanks

**Comment by MilosPaunovic** [comment]
As I can see from the `package.json` file of the library you're using ([vue-json-editor](https://github.com/dirkliu/vue-json-editor/blob/c68bfdef3b86e373a20a84778a54f370915399c4/package.json#L40)) they have `Vue@2.2.6` as a dependency, and `Quasar@2` is utilizing `Vue@3`.

So, my suggestion would be to search for Vue3 compatible library.
[https://www.npmjs.com/package/vue3-json-editor](https://www.npmjs.com/package/vue3-json-editor)

**Reply by hakuna0829** [reply]
@MilosPaunovic 
Thanks for your quick message. 
I removed the old one and installed vue3 version again.
https://www.npmjs.com/package/vue3-json-editor
I followed the readme file but still get this console issue.

`Uncaught (in promise) TypeError: Cannot read properties of null (reading 'getMode')`

In the view, there is only save button.
![image](https://user-images.githubusercontent.com/50372774/151652786-b8ee8774-c7ca-4e21-8e23-4163a0d4c497.png)

Please guide me.

**Reply by yusufkandemir** [reply]
@hakuna0829 this error is different. Please follow https://github.com/joaomede/vue3-json-editor/issues/9 for updates.

**Comment by hakuna0829** [comment]
@yusufkandemir
Thanks, but I can't find the solution there, too.
https://github.com/joaomede/vue3-json-editor/issues/9

**Reply by yusufkandemir** [reply]
I didn't state there is a solution, I just said follow it for updates, because there isn't a solution ATM. It's an external library problem, so nothing to do with Quasar.

**Reply by hakuna0829** [reply]
@yusufkandemir 
Well, is there any other package that can work with Quasar to use json editor?

**Reply by yusufkandemir** [reply]
I don't know, you don't need necessarily need to check for Quasar compatibility, any package that works with Vue 3 will work with Quasar too.

**Comment by hakuna0829** [comment]
Is there anyone who knows the useful json-editor for vue3?
